pennylane_qiskit/basic_aer.py

```diff
@@ -97,15 +47,4 @@ class BasicSimulatorDevice(QiskitDevice):
     )
 
     def __init__(self, wires, shots=1024, backend="basic_simulator", **kwargs):
-
-        min_version = Version("1.0.0")
```

Comment:
If we are removing this how do we let users know we no longer support versions prior to `1.0.0` during runtime?

Reply:
You will get an import error from now on instead from Qiskit's end since I deleted
`if Version(qiskit.__version__) >= Version("1.0.0"):`
   ` from qiskit.providers.basic_provider import BasicProvider`